test/model/src/include/model/driver/kv_workload_runner_wt.h

```diff
@@ -155,6 +159,10 @@ class kv_workload_runner_wt {
         size_t num_tables;
         table_state tables[256]; /* The table states; protected by the same lock as the URI map. */
 
+        /* WiredTiger configuration specified by the workload (protected by the connection lock). */
```

Comment:
The comment on new line 158 does not match new line 160.

Reply:
Oh, that's right! Fixed in [6ea43c7](https://github.com/wiredtiger/wiredtiger/pull/10943/commits/6ea43c73999e1936f25c20d81eb3f8004acd0a3a).